lms/djangoapps/courseware/views/views.py

```diff
@@ -1583,6 +1613,17 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
                     )
                 )
 
+        # For courses using the edx-exams service:
+        # Access to exam content is determined by edx-exams and passed the the LMS using a
+        # signed JWT. There is no longer a need for exam gating or logic inside the
+        # sequence block or its render call. As a result decendants_are_gated will not return
+        # true for these exams. Instead these sequences are gated by default and we look for
+        # an access token on the request to allow rendering to continue.
+        exam_block = ancestor_sequence_block if ancestor_sequence_block else block
+        if getattr(exam_block, 'is_time_limited', None):
```

Comment:
Is this 'is_time_limited' checking if this is a special exam? Otherwise, could you clarify what is checked here?

Reply:
yup!